### PR TITLE
Add most basic integration test

### PR DESCRIPTION
**Pull request by mellenburg**
This test expects a cluster to be provided externally and tests authentication and basic DC/OS service health

### CODE REVIEW

**Comment by colin-msphere** [comment]
Please add some documentation, either README or doc string, on how to run the integration test. Does it need a cluster already setup? How do I point the test to that cluster?

File: integration_test.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+""" Integration test (i.e. run against a real DC/OS cluster) for dcos-test-utils
+
+Note: this tests expects a DC/OS cluster to already be provisioned.
+
+At a minimum, the following environment variables should be set:
+    DCOS_DNS_ADDRESS
```
Comment (colin-msphere) [inline integration_test.py]: Why not `DCOS_CLUSTER_URL` or `DCOS_URL`? Similar names are used in [shakedown](https://github.com/dcos/shakedown#usage) and [dcos cli](https://github.com/dcos/dcos-cli#initialization).

Comment (mellenburg) [reply integration_test.py]: its historical. If you want to refactor it, that's fine, but I don't think this is the appropriate place

Comment (mellenburg) [reply integration_test.py]: If you want to see the history, this is where it came from: https://github.com/dcos/dcos/blob/1.8/packages/dcos-integration-test/extra/conftest.py#L71

**Pull request merged**